src/auto_client/__init__.py

```diff
@@ -0,0 +1,22 @@
+import requests
```

Comment:
Przenieś to do innego pliku niż __init__.py, bo ten kod będzie wołany za każdym razem przy użyciu moduły auto_client.
Możesz nazwać to "request_example.py" albo coś takiego